import-export-cli/credentials/credentials.go

```diff
@@ -85,3 +87,37 @@ func GetOAuthAccessToken(credential Credential, env string) (string, error) {
 func GetBasicAuth(credential Credential) string {
 	return Base64Encode(fmt.Sprintf("%s:%s", credential.Username, credential.Password))
 }
+
+//Revoke access Token when user is logging out from environment
+func RevokeAccessToken(credential Credential, env string, token string)  error {
+
+	//get revoke endpoint
+	tokenRevokeEndpoint := utils.GetTokenRevokeEndpoint(env,utils.MainConfigFilePath)
+	//Encoding client secret and client Id
+	var b64EncodedClientIDClientSecret = utils.GetBase64EncodedCredentials(credential.ClientId,credential.ClientSecret)
+	// set headers to request
+	headers := make(map[string]string)
+	headers[utils.HeaderContentType] = utils.HeaderValueXWWWFormUrlEncoded
+	headers[utils.HeaderAuthorization] = utils.HeaderValueAuthBasicPrefix + " " + b64EncodedClientIDClientSecret
+
+	//Create body for the request
+	body := utils.HeaderToken + token + utils.TokenTypeForRevocation
+
+	resp, err := utils.InvokePOSTRequest(tokenRevokeEndpoint, headers, body)
+	utils.Logln(utils.LogPrefixInfo + "connecting to " + tokenRevokeEndpoint)
```

Comment:
Don't we need to put this log line before the invokePostRequest method? as per the log, it should print before the request originated from the ctl.

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e